CmdStarter.Tests.Commands/Interfaces/IHasAutoComplete.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.CommandLine.Completions;
+
+namespace com.cyberinternauts.csharp.CmdStarter.Tests.Commands.Interfaces
+{
+    public interface IHasAutoComplete
+    {
+        IEnumerable<CompletionItem> OptionExpected();
```

Comment:
I think I would rename to `OptionCompletionsExpected`, because now it is like we expect `Option` objects, not `CompletionItem` objects.

(Same for the `ArgumentExpected`)

Reply:
Done.